Modules/ModParser.lua

```diff
@@ -1309,14 +1309,22 @@ for name, grantedEffect in pairs(data.skills) do
 		gemIdLookup[grantedEffect.name:lower()] = grantedEffect.id
 	end
 end
-local function extraSkill(name, level, noSupports)
+local function extraSkillGranted(name, level, noSupports)
```

Comment:
I would suggest naming these `triggerExtraSkill` and `grantExtraSkill` to prevent the wrong one from being used.

Reply:
Done